.github/workflows/IntegrationTest.yml

```diff
@@ -0,0 +1,57 @@
+name: IntegrationTest
+
+on:
+  push:
+    branches: [master]
+    tags: [v*]
+  pull_request:
+
+concurrency:
+  # Skip intermediate builds: always.
+  # Cancel intermediate builds: only if it is a pull request build.
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ startsWith(github.ref, 'refs/pull/') }}
+
+jobs:
+  test:
+    name: ${{ matrix.package.repo }}/${{ matrix.package.group }}
+    runs-on: ubuntu-latest
+    env:
+      GROUP: ${{ matrix.package.group }}
+    strategy:
+      fail-fast: false
+      matrix:
+        package:
+          - {user: JuliaDiff, repo: AbstractDifferentiation.jl, group: All}
```

Comment:
Maybe we could comment out the ones that fail for now? Or make this integration test optional for the green CI build badge?

Reply:
I think it would be good to keep the tests nevertheless. It's a separate workflow so it won't affect the CI badge.

Reply:
right I missed that